### PR TITLE
Add Nextra Detection

### PR DESCRIPTION
**Pull request by ysm-dev**
https://nextra.vercel.app/

- https://nextra.vercel.app/
- https://swr.vercel.app/

### CODE REVIEW

File: src/technologies/n.json [hunk 1]
```diff
@@ -658,6 +658,19 @@
     "scriptSrc": "(?:api|sdk)\\.nextsale\\.io/",
     "website": "https://nextsale.io"
   },
+  "Nextra": {
+    "cats": [
+      57
+    ],
+    "description": "Nextra is Next.js based static site generator.",
+    "dom": "[class*='.nextra-container']",
```
Comment (nurbek91) [inline src/technologies/n.json:666]: Maybe we will check one or a couple of tags
`dom": "div.nextra-container",
`
I also see a js object in one of the examples.
As I understand it is less resource intensive to check the js object
```
"js": {
      "__nextra_internal__": ""
    },
```

File: src/technologies/n.json [hunk 2]
```diff
@@ -658,6 +658,19 @@
...
+    "icon": "Next.js.svg",
+    "implies": [
+      "Next.js"
+    ],
```
Comment (nurbek91) [inline src/technologies/n.json:670]: Can be changed to
`"requires": "Next.js",`
Next.js seems to be defined everywhere. If we replace with required, then we will not check classes on all other sites

**Review by nurbek91 — COMMENTED** [review]
Hey mate, little changes needed from my point of view

File: src/technologies/n.json (hunk 1 above)
Comment (ysm-dev) [reply src/technologies/n.json]: [36101b1](https://github.com/wappalyzer/wappalyzer/pull/7127/commits/36101b1f0a4838a687e11d85af5fece6ac77ce24)

[eb70dfc](https://github.com/wappalyzer/wappalyzer/pull/7127/commits/eb70dfc97b0319475b1c6de4b102aa00c073322d)

File: src/technologies/n.json (hunk 2 above)
Comment (ysm-dev) [reply src/technologies/n.json]: [8a0c3c5](https://github.com/wappalyzer/wappalyzer/pull/7127/commits/8a0c3c51dd165318487c36ff9aff27198beb122a)

**Review by nurbek91 — APPROVED** [review]
thanks mate!!

**Pull request merged**